### ISSUE TITLE
JTCOP is not connected

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#398]
Now its perfectly legal to do this:

```java
@Test
void boom() {       
}
```

It should be prohibited as well other test bad practices. Let's connect https://github.com/volodya-lombrozo/jtcop

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel I agree, let's do it
